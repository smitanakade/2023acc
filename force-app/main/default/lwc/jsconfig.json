{
    "compilerOptions": {
        "experimentalDecorators": true,
        "checkJs": true,
        "target": "ESNext",
        "lib": "ESNext",
        "baseUrl": "./",
        "paths": {
            "c/*": [
                "*/"
            ]
        },
        "moduleResolution": "node"
    },
    "include": [
        "**/*",
        "../../../../.sfdx/typings/lwc/**/*.d.ts",
        "../../../../types/**/*.d.ts"
    ],
    "exclude": [
        "cf**/*",
        "node_modules"
    ],
    "typeAcquisition": {
        "include": [
            "jest"
        ]
    },
    "paths": {
        "c/*": [
            "*"
        ]
    }
}
