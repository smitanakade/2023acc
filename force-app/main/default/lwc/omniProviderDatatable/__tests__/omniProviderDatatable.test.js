import { createElement } from "lwc";
import OmniProviderDatatable from "c/omniProviderDatatable";

describe("c-omni-provider-datatable", () => {
  afterEach(() => {
    // The jsdom instance is shared across test cases in a single file so reset the DOM
    while (document.body.firstChild) {
      document.body.removeChild(document.body.firstChild);
    }
  });

  it("matches snapshot", () => {
    // Arrange
    const element = createElement("c-omni-provider-datatable", {
      is: OmniProviderDatatable,
    });

    document.body.appendChild(element);

    expect(element).toMatchSnapshot();
  });
});
