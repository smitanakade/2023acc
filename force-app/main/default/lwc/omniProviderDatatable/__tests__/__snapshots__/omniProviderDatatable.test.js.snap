// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-omni-provider-datatable matches snapshot 1`] = `
<c-omni-provider-datatable>
  #shadow-root(open)
</c-omni-provider-datatable>
`;
